modin/experimental/engines/pandas_on_ray/io_exp.py

```diff
@@ -53,6 +57,14 @@ def _read_parquet_columns(path, columns, num_splits, kwargs):  # pragma: no cove
 
 
 class ExperimentalPandasOnRayIO(PandasOnRayIO):
+    build_args = dict(
+        frame_partition_cls=PandasOnRayFramePartition,
+        query_compiler_cls=PandasQueryCompiler,
+        frame_cls=PandasOnRayFrame,
+    )
+    read_csv = type(
```

Comment:
Can we rename this to avoid confusion? Or do you disagree with this?
```suggestion
    read_csv_glob = type(
```

Reply:
Sure that sounds good. I have made the change